tensorflow/core/kernels/segment_reduction_ops_gpu.cu.h

```diff
@@ -29,8 +30,15 @@ limitations under the License.
 #include "tensorflow/core/util/env_var.h"
 #include "tensorflow/core/util/gpu_device_functions.h"
 #include "tensorflow/core/util/gpu_kernel_helper.h"
+#include "tensorflow/core/util/gpu_solvers.h"  // For ScratchSpace
 #include "tensorflow/core/util/permutation_input_iterator.h"
 
+#if GOOGLE_CUDA
```

Comment:
We usually check ` (defined(GOOGLE_CUDA) && GOOGLE_CUDA) ` to avoid compiler warnings when the symbol isn't defined.  Same with ROCM.

Reply:
It's already used like this in [the same file](https://github.com/benbarsdell/tensorflow/blob/12e522e161c39cb4c89d1be152f392b80db469f4/tensorflow/core/kernels/segment_reduction_ops_gpu.cu.h#L19C39-L19C39) as well as hundreds of other places in the codebase.